src/script/util/Environment.ts

```diff
@@ -33,7 +33,7 @@ const getAppVersion = (): string => {
 
 const getElectronVersion = (userAgent: string): string => {
   // [match, version]
-  const [, electronVersion] = /Wire(?:Internal)?\/(\S+)/.exec(userAgent) || [];
+  const [, electronVersion] = /Wire(?:Internal)?\/(\S+)/i.exec(userAgent) || [];
```

Comment:
Now that we know it's always lowercase we could do:
```suggestion
  const [, electronVersion] = /wire(?:internal)?\/(\S+)/.exec(userAgent) || [];
```

Reply:
Why?